src/release/breaking-changes/windows-build-architecture.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Windows build path changed to add the target architecture
+description: >
+  In preparation for supporting Windows Arm64, 
+  the Windows build path was updated to include the target architecture.
+---
+
+## Summary
+Flutter Window's build path was updated to include the target architecture.
+
+## Context
+In preparation for supporting Windows Arm64, the Windows build path was
+updated to add the build's target architecture.
+
+Previously, Flutter Windows build assumed an x64 target architecture.
+
+## Migration guide
+You may need to update your infrastructure to use the new Flutter Windows
+build path.
+
+Example build path before the migration:
+`build\windows\runner\Release\hello_world.exe`
+
+Example build path after the migration:
+`build\windows\x64\runner\Release\hello_world.exe`
+
+If you use [`package:msix`][], update to version 3.15.0 or newer.
+
+[`package:msix`]: https://pub.dev/packages/msix
+
+## Timeline
+Landed in version: TBD
```

Comment:
The commit isn't tagged in a release yet. What's the best way to track this so that I can update this when ready? 

See: https://github.com/flutter/flutter/commit/792e26df9540cf2cf510ea0c11a53ba10a243424

Reply:
We try not to land the breaking change/migration guides before their related commits are tagged, unless it's pressing. This reduces the chance of reverts we aren't aware of or other quick follow-up changes that would change the doc.

Then I usually check open PRs most days to see if they're tagged. If they are, I add the version tag myself and merge the PR. It works for now, but we might explore something more automatic based off the commit in the future. 

Overall, you don't need to worry about updating it. If you'd like to see this published before then though, let us know :)

Reply:
Sounds good, thanks for the detailed information!